lib/synx/xcodeproj_ext/project/object/pbx_group.rb

```diff
@@ -31,7 +39,11 @@ def move_entries_not_in_xcodeproj
           else
             Synx::Tabber.puts "#{basename}/".green
             Synx::Tabber.increase
-            Dir[real_path.to_s + "/{*,.*}"].reject { |e| %W(. ..).include?(Pathname(e).basename.to_s) }.each do |entry|
+            Dir[real_path.to_s + "/{*,.*}"]
+            .reject { |e| %W(. ..).include?(Pathname(e).basename.to_s) }
+            .each do |entry|
+              # Is this right? entry should be an absolute path here, so it should
```

Comment:
Ah! Ok then, I replaced this with a clearer equivalent version, see if that looks good.